common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3941,6 +3947,12 @@
     ],
     "sqlState" : "42P20"
   },
+  "MULTIPLE_PRIMARY_KEYS" : {
+    "message" : [
+      "Multiple primary keys are defined. Please ensure that only one primary key is defined for the table."
```

Comment:
Can we name the primary key? Like: A primary key <keyName> already exists for <tableName>. Only one primary key can be created per table.

Reply:
Good question. What if there are multiple primary keys in a create table statement
```
CREATE TABLE t(
  i int primary key,
  j int primary key)
```

It doesn't make sense to name that `i` already exists

Reply:
I guess we can include conflicting lists of columns for which we are trying to define PKs?